app/src/main/java/org/fossasia/susi/ai/adapters/viewholders/MapViewHolder.java

```diff
@@ -20,18 +31,75 @@
     @BindView(R.id.map_image)
```

Comment:
Change it to `public class mapViewHolder extends RecyclerView.ViewHolder` from `public class MapViewHolder extends MessageViewHolder`